eleventy-config.js

```diff
@@ -21,6 +21,7 @@ module.exports = function(eleventyConfig) {
 
   eleventyConfig.addPlugin(pluginRss);
   eleventyConfig.addFilter('length', getFileLength);
+  eleventyConfig.addFilter('getReadingListEntry', getReadingListEntry);
```

Comment:
How does this work without an `addCollection` call?

Reply:
i'm not 100% sure how all the data is scoped, but filters can take arguments, so i just pass the whole reading list into it in the template.